tasks/fix-cat2.yml

```diff
@@ -1098,8 +1098,9 @@
   block:
       - name: "MEDIUM | RHEL-07-021030 | AUDIT | All world-writable directories must be group-owned by root, sys, bin, or an application group."
         command: find {{ rhel7stig_local_mounts | join(' ') }} -xdev -type d -perm -002 -gid +999
-        changed_when: no
-        register: rhel_07_021030_audit.stdout != ""
+        changed_when: rhel_07_021030_audit.stdout != ""
+        check_mode: "{{rhel7stig_disruptive_check_mode}}"
```

Comment:
this should be `check_mode: no`

`{{ rhel7stig_disruptive_check_mode }}` should go on the following task that would actually make the change

Reply:
Ok. I must have misread the variable description and the other (single?) example to go from. Will fix these in the morning.